src/calculateRentalCost.js

```diff
@@ -3,8 +3,25 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const dayRentCost = 40;
```

Comment:
The constant 'dayRentCost' is descriptive and uses 'const' which is good practice to prevent unintended reassignments.